### ISSUE TITLE
UnicodeEncodeError: 'charmap' codec can't encode character '\u03b2' in position 6: character maps to <undefined> [17728] Failed to execute script '__main__' due to unhandled exception!

### ISSUE THREAD

**Issue by Ivan-vechetti** [Metaboverse/Metaboverse#128]
Hi, I am not sure this is a bug, but I am encountering this error: 
UnicodeEncodeError: 'charmap' codec can't encode character '\u03b2' in position 6: character maps to <undefined>
[17728] Failed to execute script '__main__' due to unhandled exception!. What does that mean?

Thanks

**Comment by oolonek** [comment]
Getting similar errors in the logs.

```

45
46 ########
47 STDERR:
48 ########
49   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
50                                  Dload  Upload   Total   Spent    Left  Speed
51 ^M  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0^M  0 41.3M    0 47920    0     0  52544      0  0:13:44 --:--:--  0:13:44 52543^M  1 41.3M    1  468k    0     0   238k      0  0:02:57  0:00:01  0:02:56  238k>
52 Traceback (most recent call last):
53   File "__main__.py", line 246, in <module>
54   File "__main__.py", line 228, in main
55   File "analyze/__main__.py", line 292, in __main__
56   File "analyze/__main__.py", line 102, in process_data
57   File "analyze/prepare_data.py", line 410, in __main__
58   File "analyze/prepare_data.py", line 231, in catenate_data
59   File "analyze/prepare_data.py", line 231, in <listcomp>
60   File "analyze/prepare_data.py", line 38, in eval_table
61   File "pandas/core/frame.py", line 8933, in applymap
62   File "pandas/core/frame.py", line 8848, in apply
63   File "pandas/core/apply.py", line 733, in apply
64   File "pandas/core/apply.py", line 857, in apply_standard
65   File "pandas/core/apply.py", line 873, in apply_series_generator
66   File "pandas/core/frame.py", line 8931, in infer
67   File "pandas/_libs/lib.pyx", line 2870, in pandas._libs.lib.map_infer
68   File "analyze/prepare_data.py", line 38, in <lambda>
69   File "ast.py", line 110, in literal_eval
70   File "ast.py", line 109, in _convert
71   File "ast.py", line 83, in _convert_signed_num
72   File "ast.py", line 74, in _convert_num
73   File "ast.py", line 71, in _raise_malformed_node
74 ValueError: malformed node or string on line 1: <ast.Name object at 0x7f3f2899e800>
75 [1467769] Failed to execute script '__main__' due to unhandled exception!

````

**Comment by Ivan-vechetti** [comment]
I have figured it out. In my case was a Greek letter that I had to remove.

**Comment by oolonek** [comment]
Well it doesn't seems to be the case in the input I am using ... feel free to reopen

**Comment by oolonek** [comment]
Letting this here for the record.
The issue was that I had an entry with an  `Inf` string instead of a float.

**Comment by j-berg** [comment]
Hi @oolonek, thanks for raising this issue. Others are running into similar issues. I am going to add this to my to-do list for the next release. I am going to re-open to track these changes, but the updates I am hoping to implement are related to #129.

**Comment by j-berg** [comment]
Fixes scheduled for Metaboverse v0.10.2 (see #129 )
